tests/manual_tests/visual_screen_tests/visual_screen_tests.py

```diff
@@ -273,7 +591,9 @@ def stall_jig_screen_tests():
 
             set_up_dummy_serial(status, alarm_message)
 
-            # CHANGE ME
+            stall_jig_screen = screen_stall_jig.StallJigScreen(name='stall_jig', systemtools = systemtools_sm, machine = m, job = jd, settings = sett, localization = l, calibration_db = db)
```

Comment:
My guess is that this is because stall jig init function sets non kwarg attributes which messes with the screen maker's scraping functionality. As it's Archie's code and I don't understand it, I decided it would be quicker to just leave it to use regular initialisation (it's the only screen in the test anyway)